Intersect.Client/Networking/PacketHandler.cs

```diff
@@ -2121,6 +2121,18 @@ public void HandlePacket(IPacketSender packetSender, GuildInvitePacket packet)
             );
         }
 
+        public void HandlePacket(IPacketSender packetSender, FadePacket packet)
+        {
+            if (packet.FadeOut)
+            {
+                Fade.FadeOut(true);
+            }
+            else
+            {
+                Fade.FadeIn(true);
+            }
```

Comment:
With my enum this could be a switch and `None` would call `Cancel`

Reply:
Done